### DISCUSSION TITLE
Support for `data:` URL

### DISCUSSION THREAD

**Discussion by johnnyshankman** [node-fetch/node-fetch#1689] [Ideas]
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs

This works in most browsers now.

**Comment by jimmywarting** [comment]
Uhm... node-fetch already do support data: urls

**Comment by johnnyshankman** [comment]
Last time I checked this was not the case. Can you provide a codepen or something proving this works? I was forced to switch fetch providers to get this working back in dec 2022. I'd do it myself but codepen in browser with node is not a thing!

I found `fetch` works but `node-fetch` errors on JSON Data URIs using base64 or utf8.
